GothicVRProject/GothicVR
# Feature/vob particlefx

This PR is intended to merge first Particle implementation. It's definitely WIP (e.g. fire) but the creation flow is fine. I want to merge it and will work on it at a later stage again.

# To test
1. Activate FeatureFlag: _enableVobParticles_ (under Experimental)
2. Check if you see properly flying leaves inside woods
4. Check if you see broken fire particles (expected)


# Checklist
(Please judge for yourself which entry is valuable for your PR)

## Scene

# Testing

## Dependencies
